### ISSUE TITLE
Remove the state of pods and container before creating the diskimage

### ISSUE THREAD

**Issue by praveenkumar** [code-ready/snc#22]
We need to remove all the pods/container before creating the disk images so that when it is used for crc purpose it will not take the state of cluster which is shutdown. This should be part of create disk since it will be executed just before that or we can create an another script which contains the steps like deployments/replicas changes and this one and executed before `createdisk` but after the cert rotation from the snc side.

=== On the VM ==

``` 
$ systemctl stop kubelet
$ crictl stopp $(crictl pods -q)
$ crictl rmp $(crictl pods -q)
```

**Comment by anjannath** [comment]
@praveenkumar The steps you mentioned are the ones required to clear the state of pods, or are there more things that needs to be done?

**Comment by praveenkumar** [comment]
fixed by #43
